soc/nxp/mcx/mcxn/soc.c

```diff
@@ -26,9 +26,9 @@ void soc_reset_hook(void)
 
 #endif
 
-#define FLEXCOMM_CHECK_2(n)	\
-	BUILD_ASSERT((DT_NODE_HAS_COMPAT(n, nxp_lpuart) == 0) &&		\
-		     (DT_NODE_HAS_COMPAT(n, nxp_lpi2c) == 0),			\
+#define FLEXCOMM_CHECK_2(n)                                                                        \
```

Comment:
not a blocking comment but the changes here really don't seem useful, is this clangformat or manually applied or what?

Reply:
Yes clang-format...

Reply:
Was not requesting a change here just that the change has no relevance to the commit, generally a clangformat change would be in it's own commit

Reply:
Good remark. I will do it.